### PR TITLE
start metricsCollector,eventGenerator and scalingEngine in integration test

### PR DESCRIPTION
**Pull request by qibobo**
start metricsCollector,eventGenerator and scalingEngine in integration test

### CODE REVIEW

File: src/integration/integration_metricscollector_eventgenerator_scalingengine_test.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+		scalingEngineConfPath = prepareScalingEngineConfig(dbUrl, components.Ports[ScalingEngine], fakeCCNOAAUAA.URL(), cf.GrantTypePassword)
+		startMetricsCollector()
+		startEventGenerator()
+		startScalingEngine()
+	})
+
+	AfterEach(func() {
+		stopAll()
+	})
+	Describe("", func() {
```
Comment (fraenkel) [inline src/integration/integration_metricscollector_eventgenerator_scalingengine_test.go]: Where are the tests?

File: src/integration/integration_suite_test.go [hunk 2]
```diff
@@ -236,6 +281,123 @@ autoscaler.scalingengine.url=%s
 	return cfgFile.Name()
 }
 
+func prepareMetricsCollectorConfig(dbUri string, port int, ccNOAAUAAUrl string, cfGrantTypePassword string) string {
```
Comment (fraenkel) [inline src/integration/integration_suite_test.go]: Shouldn't this live in components.go?

Comment (qibobo) [inline src/integration/integration_suite_test.go]: Moved the prepare config functions to components.

File: src/integration/integration_metricscollector_eventgenerator_scalingengine_test.go (hunk 1 above)
Comment (qibobo) [inline src/integration/integration_metricscollector_eventgenerator_scalingengine_test.go]: For this pull request I just put the start components functions here. So all the integration test can use these functions.

**Comment by qibobo** [comment]
Add the test cases.

**Pull request merged**